src/components/Index/BillContent.svelte

```diff
@@ -15,13 +15,53 @@
 		bills: Pick<Bill, 'id' | 'title' | 'nickname'>[];
 	}
 
+	type MpTerm = {
+		name: string;
+		custom_name?: string;
+		founding_date: string | null;
+		dissolution_date: string | null;
+		term: number | null;
+	};
+
+	let mpterm_display_name = (mp: MpTerm) => {
```

Comment:
Better write function name as verb, so it will be easy to know which one is function, which one is variable. e.g. `getMptermDisplayName`.